ccd-definition/FixedLists.json

```diff
@@ -262,5 +262,59 @@
     "ID": "Consent",
     "ListElementCode": "agree",
     "ListElement": "I agree with this statement"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentsLitigationList",
+    "ListElementCode": "Yes",
+    "ListElement": "Yes"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentsLitigationList",
+    "ListElementCode": "No",
+    "ListElement": "No"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentsLitigationList",
+    "ListElementCode": "Don't know",
```

Comment:
Fixed list codes are not for human. Make it more like enum in Java, something we can easily compare in Java code.

Of course that will require mapping in PDF template to human readable text.